autogpt_platform/backend/backend/server/v2/library/routes/presets.py

```diff
@@ -401,6 +457,14 @@ async def execute_preset(
     merged_node_input = preset.inputs | inputs
     merged_credential_inputs = preset.credentials | credential_inputs
 
+    for page in range(1, 10):
```

Comment:
I don't. get when you remove 10 pages and remove 5 pages for others, what's the factor?